00_Base/src/config/types.ts

```diff
@@ -103,8 +104,11 @@ export const systemConfigSchema = z.object({
         })
     }),
     websocketServer: z.object({
+        webProtocol: z.enum(['http', 'https']).default('http'),
```

Comment:
@thanaParis This should be an `TLS flag` instead of `webProtocol`. If we use something like `webProtocol` it should be named `socketProtocol` and would require values like `WSS` or `WS`.